File: src/app/models/core/spells/castingTime.ts
export class CastingTime {
	public id: number;
	public amount: number;
	public unit: string;
	public reactionCondition: string;

	public static getDefault(): CastingTime {
		const time = new CastingTime();
		time.id = -1;

		return time;
	}
}

File: api/Ganymede.Api.Data/Spells/CastingTime.cs
using System.Collections.Generic;

namespace Ganymede.Api.Data.Spells
{
    public class CastingTime
    {
        public int ID { get; set; }
        public int Type { get; set; }
        public int Amount { get; set; }
        public string Unit { get; set; }
        public string ReactionCondition { get; set; }

        public virtual ICollection<SpellCastingTime> Spells { get; set; }
    }
}
